src/gui/virtualstudio.cpp

```diff
@@ -1363,6 +1377,15 @@ void VirtualStudio::getServerList(bool firstLoad, bool signalRefresh, int index)
             if (yourServers.isEmpty()) {
                 if (subServers.isEmpty()) {
                     m_logoSection = QStringLiteral("Public Studios");
+
+                    if (pubServers.isEmpty() && skippedStudios == 0) {
+                        // This is a new user
+                        setShowCreateStudio(true);
```

Comment:
If user A just happens to have one of their public studios running and user B is a new user logging in for the first time, then user B won't see the new user screen. Do I have that right?

Just want to make sure I understand that my understanding is correct. (Also recognizing that this is largely intended to be serve as a patch so that we can shut down JackTrip Public Studios if a more robust solution isn't achievable right away)

Reply:
Yes. this is catching an edge case, not creating a FTUX.

Reply:
cool, thanks for the clarification